### PR TITLE
feat: websockets and events changes

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: internal/app/api/v1/events.go [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+			if resp.Error() != "" {
+				s.Log.Errorw("got error when sending webhooks", "response", resp)
+				continue
+			}
+			s.Log.Debugw("got event response", "response", resp)
+		}
+	}()
+}
+
+func (s TestkubeAPI) EventsTestHandler() fiber.Handler {
```
Comment (vsukhin) [inline internal/app/api/v1/events.go:36]: Is this for testing?

Comment (exu) [reply internal/app/api/v1/events.go]: yes, removed.

File: pkg/event/emitter.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	"go.uber.org/zap"
+
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+	"github.com/kubeshop/testkube/pkg/event/kind/common"
+	"github.com/kubeshop/testkube/pkg/log"
+)
+
+const (
+	eventsBuffer      = 10000
+	workersCount      = 20
```
Comment (vsukhin) [inline pkg/event/emitter.go:17]: should it be harcoded or config params?

File: pkg/event/kind/slack/listener.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	err := slacknotifier.SendEvent(event.Type_, *event.Execution)
+	if err != nil {
+		return testkube.NewFailedEventResult(event.Id, err)
+	}
+
+	return testkube.NewSuccessEventResult(event.Id, "event sent to slack")
+}
+
+func (l *SlackListener) Kind() string {
+	return "slack"
```
Comment (vsukhin) [inline pkg/event/kind/slack/listener.go:48]: Typed value?

**Review by vsukhin — APPROVED** [review]
looks good

**Pull request merged**